i18n/zh-cn/user-docs/installation/install-with-helm/install-kubeblocks-with-helm.md

```diff
@@ -95,6 +95,125 @@ KubeBlocks 是基于 Kubernetes 的原生应用，你可以使用 Helm 来进行
+   ## @param image.pullPolicy KubeBlocks image pull policy
+   ## @param image.tag KubeBlocks image tag (immutable tags are recommended)
+   ## @param image.imagePullSecrets KubeBlocks image pull secrets
+   ## @param image.tools.repository KubeBlocks tools image repository
+   image:
+     registry: registry.kb  //换成可用的 registry
+     repository: apecloud/kubeblocks
+     pullPolicy: IfNotPresent
+     # Overrides the image tag whose default is the chart appVersion.
+     tag: ""
+     imagePullSecrets: []
+     tools:
+       repository: apecloud/kubeblocks-tools
+     datascript:
+       repository: apecloud/kubeblocks-datascript
+    
+   
+   
+   ## @param addonChartsImage - addon charts image, used to copy Helm charts to the addon job container.
+   ## @param addonChartsImage.chartsPath - the helm charts path in the addon charts image.
+   addonChartsImage:
+     # if the value of addonChartsImage.registry is not specified using `--set`, it will be set to the    value of 'image.registry' by default
+     registry: "registry.kb"  // 换成可用的 registry
+     repository: apecloud/kubeblocks-charts
+     pullPolicy: IfNotPresent
+     tag: ""
+     chartsPath: /charts
+     pullSecrets: []
+   ```
+4. 将需要的远程镜像重新 tag 推到 K8S 的镜像仓库
+   1. 基本需要的镜像是下面这几个
+   ```bash
+   infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/kubeblocks-datascript:0.8.4-beta.11
+   infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/kubeblocks-tools:0.8.4-beta.11
+   infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/kubeblocks:0.8.4-beta.11
+   infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/snapshot-controller:v6.2.1
+   infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/kubeblocks-charts:0.8.4-beta.11
+   ```
+   2. 将他们重新 tag 后推到远端（以kubeblocks:0.8.4-beta.11举例）
+   ```bash
+   docker pull infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/kubeblocks:0.8.4-beta.11
+   docker tag infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/kubeblocks:0.8.4-beta.11 registry.kb/apecloud/kubeblocks:0.8.4-beta.11
+   docker push registry.kb/apecloud/kubeblocks:0.8.4-beta.11
+   ```   
+   后续有需要的镜像也可以按照上面的方式添加
+
+5. 可以将部分自动安装的 addon 关闭，只保留需要的
+
+   在kubeblocks/templates/addons 文件夹中，将对应的数据库的配置中的 autoInstall 改成 false（以 apecloud-mysql-addon.yaml举例）
+   ```bash
+   {{- $selectorLabels := include "kubeblocks.selectorLabels" . }}
+   {{- include "kubeblocks.buildAddonCR" (merge (dict
+     "kbVersion" ">=0.7.0"
+     "selectorLabels" $selectorLabels
+     "name" "apecloud-mysql"
+     "version" "0.8.0-beta.8"
+     "model" "RDBMS"
+     "provider" "apecloud"
+     "description" "ApeCloud MySQL is a database that is compatible with MySQL syntax and achieves high    availability through the utilization of the RAFT consensus protocol."
+     "autoInstall" false ) .) -}}  // 将原本的 true 改成false
+   ``` 
+6. 部署
+   
+   将原本的 kubeblocks 文件夹打包上传至集群部署节点
+   ```bash
+   tar czvf kb0.8.4beta11.tgz kubeblocks
```

Comment:
The first command is executed in your local machine that can access the Internet; the second command is executed in your server in the air-gapped environment.  They shouldn't be put into the same code block.